src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/organizations.py

```diff
@@ -408,14 +426,14 @@ def build_account_path(self, ou_id, account_path, cache):
         # While not at the root of the Organization
         while current.get("Type") != "ROOT":
             # check cache for ou name of id
-            if not cache.exists(current.get("Id")):
-                cache.add(
+            if not self.cache.exists(current.get("Id")):
+                self.cache.add(
                     current.get("Id"),
                     self.describe_ou_name(current.get("Id")),
                 )
-            ou_name = cache.get(current.get("Id"))
+            ou_name = self.cache.get(current.get("Id"))
```

Comment:
We don't need to cache this at this layer anymore, as you added caching in the describe_ou_name function 👍 

Hence, this would suffice:
```suggestion
            ou_name = self.describe_ou_name(current.get("Id"))
```